api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2987,19 +2987,27 @@ public void removeObject(String bucketName, String objectName)
   /**
    * Removes multiple objects from a bucket.
    *
+   * <p><b>Note:</b></p>
+   * This method does lazy evaluation. You need to traverse over the returned Iterable to do the
+   * real delete jobs.
```

Comment:
You would need to modify description than adding new style here. Below is an idea.
```
Removes multiple objects from a bucket. As objects removal are lazily executed, its required to iterate returned Iterable.
```

Reply:
ok